src/allmydata/test/cli/test_backup.py

```diff
@@ -407,12 +421,16 @@ def test_exclude_from_tilde_expansion(self):
 
         ns = Namespace()
         ns.called = False
+        original_open = open
         def call_file(name, *args):
-            ns.called = True
-            self.failUnlessEqual(name, abspath_expanduser_unicode(exclude_file))
-            return StringIO()
-
-        patcher = MonkeyPatcher((__builtin__, 'file', call_file))
+            if name.endswith("excludes.dummy"):
+                ns.called = True
+                self.failUnlessEqual(name, abspath_expanduser_unicode(exclude_file))
+                return StringIO()
+            else:
+                return original_open(name, *args)
+
+        patcher = MonkeyPatcher((builtins, 'open', call_file))
```

Comment:
Oy.... okay.

Reply:
It used to rely on the fact that most places used `open()` but just this one place used `file()`, so...